From 86d4723b09d36d0d14141f4f735207a761994a72 Mon Sep 17 00:00:00 2001
From: Yongjun Zhang <yzhang@cloudera.com>
Date: Mon, 27 Feb 2017 20:42:13 -0800
Subject: [PATCH 2311/2848] HADOOP-14127. Add log4j configuration to enable
 logging in hadoop-distcp's tests. (Xiao Chen via
 Yongjun Zhang)

(cherry picked from commit d235dcdf0b6f36162280e32c7c0342b7970c9b7b)
(cherry picked from commit 16ca70553cd88179ceb38871fd65a8d8172984f7)

Change-Id: I36b0c19a8c8ed5e338306c7cefa952f5477e848d
---
 .../src/test/resources/log4j.properties            |   22 ++++++++++++++++++++
 1 file changed, 22 insertions(+)
 create mode 100644 hadoop-tools/hadoop-distcp/src/test/resources/log4j.properties

diff --git a/hadoop-tools/hadoop-distcp/src/test/resources/log4j.properties b/hadoop-tools/hadoop-distcp/src/test/resources/log4j.properties
new file mode 100644
index 0000000..22990ca
--- /dev/null
+++ b/hadoop-tools/hadoop-distcp/src/test/resources/log4j.properties
@@ -0,0 +1,22 @@
+#
+#   Licensed to the Apache Software Foundation (ASF) under one or more
+#   contributor license agreements.  See the NOTICE file distributed with
+#   this work for additional information regarding copyright ownership.
+#   The ASF licenses this file to You under the Apache License, Version 2.0
+#   (the "License"); you may not use this file except in compliance with
+#   the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# log4j configuration used during build and unit tests
+
+log4j.rootLogger=debug,stdout
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c{2} (%F:%M(%L)) - %m%n
-- 
1.7.9.5

